std_msgs/Header header
DynamicObjectWithFeature[] feature_objects
